docs/api/ref/schemas/product_ecoscore.yaml

```diff
@@ -99,7 +110,10 @@ properties:
             properties:
               labels:
                 type: array
-                example: "vegan, fat free, Kosher"
+                examples:
+                  - - vegan
+                    - fat free
+                    - Kosher]
```

Comment:
```suggestion
                    - Kosher
```

Reply:
sorry Teolemon, I made a new commit instead of accepting your suggestion. I still have to get used to github